components/throttle.js

```diff
@@ -0,0 +1,93 @@
+// throttle.js
+
+var _ = require('underscore');
+var fs = require('fs');
+
+var idIndex = 0;
```

Comment:
Expectation is that the throttle and throttle-fire are singletons that control the feeding of data in the network.  I wrote this because I don't want to flood the load balancer with a couple hundred thousand patient records to process.  This lets us take a batch, process, and add more when we are ready for it.